src/proc.h

```diff
@@ -28,6 +28,12 @@ struct mem_map *create_maps(int pid);
 int print_proc_maps(int pid);
 
 /*
+ * simple routine to print process comm for
+ * debugging or advanced error reporting
+ */
+int print_proc_comm(int pid);
```

Comment:
If we will decide to polish output, probably this should return number of bytes written or pointer to static data. Latter makes it convenient to pass value to `printf()`.